### PR TITLE
support mod_dub

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

File: lib/synth-audio.js [hunk 1]
```diff
@@ -619,6 +619,8 @@ const synthElevenlabs = async(logger, {credentials, options, stats, language, vo
   if (!process.env.JAMBONES_DISABLE_TTS_STREAMING && !renderForCaching) {
     let params = '';
     params += `{api_key=${api_key}`;
+    params += ',vendor=elevenlabs';
+    params += ',voice=${voice}';
```
Comment (davehorton) [inline lib/synth-audio.js:623]: I dont understand this...elevenlabs uses model_id instead of voice doesn't it?

Comment (xquanluu) [reply lib/synth-audio.js:623]: NO elevenlabs has 
voice= pNInz6obpgDQGcFmaJgB
language=en
model=turbo_v2

**Comment by davehorton** [comment]
@xquanluu does this include the changes for azure tts streaming as well?

**Comment by xquanluu** [comment]
For azure_tts_streaming, it's is here https://github.com/jambonz/speech-utils/pull/59/files

But due to new changes from you for not calling synthMicrosoft twice, Please wait I have to test this again.

**Comment by davehorton** [comment]
lets merge this into this PR as well if that makes sense

**Comment by xquanluu** [comment]
yes it make senses, let me finish testing this PR with mod_dub, I will cherry-pick to this PR after that.

**Review by davehorton — APPROVED** [review]

**Comment by davehorton** [comment]
@xquanluu please mark ready for review

**Pull request merged**